### PR TITLE
feat(jvmid): enhance UI hints when JVM ID cannot be generated

### PR DESCRIPTION
**Pull request by andrewazores**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [x] Linked a relevant issue which this PR resolves
* [x] Linked any other relevant issues, PR's, or documentation, if any
* [x] Resolved all conflicts, if any
* [x] Rebased your branch PR on top of the latest upstream `main` branch
* [x] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [x] [Signed all commits using a GPG signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#gpg-commit-signature-verification)

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

Related to cryostatio/cryostat#1695

## Description of the change:
Updates some user strings and UI elements following docs feedback

### CODE REVIEW

File: src/app/Topology/Shared/utils.tsx [hunk 1]
```diff
@@ -41,26 +50,41 @@ export const getStatusTargetNode = (node: TargetNode | EnvironmentNode): [NodeSt
...
+                  <Button variant="link" isSmall style={{ padding: 0 }}>
+                    add credentials
+                  </Button>
+                  .
+                </WarningResolverAsCredModal>
+              </Text>,
+              <Text key="jmx-ssl" component={TextVariants.p}>
+                If{' '}
+                <DescriptionListTermHelpText>
+                  <Popover maxWidth="40rem" headerContent="JMX over SSL" bodyContent={<JmxSslDescription />}>
```
Comment (mwangggg) [inline src/app/Topology/Shared/utils.tsx:78]: JMX over SSL -> SSL enabled for JMX?

File: src/app/Topology/Shared/utils.tsx [hunk 2]
```diff
@@ -41,26 +50,41 @@ export const getStatusTargetNode = (node: TargetNode | EnvironmentNode): [NodeSt
...
+            title: 'Failed to generate the JVM identifier',
             description: (
               <>
-                <Text component={TextVariants.p}>
-                  If target has JMX Authentication enabled, add the credential to Cryostat keyring.
-                </Text>
-                <Text component={TextVariants.p}>
-                  If the target has SSL enabled over JMX, add its certificate to Cryostat truststore.
-                </Text>
+                <Text component={TextVariants.p}>Check the target authentication settings:</Text>
```
Comment (mwangggg) [inline src/app/Topology/Shared/utils.tsx:56]: add the 'If JMX authentication is enabled, add credentials. If SSL is enabled for JMX, add the SSL certificate.' text?

File: src/app/Topology/Shared/utils.tsx (hunk 1 above)
Comment (andrewazores) [reply src/app/Topology/Shared/utils.tsx:78]: I think it works well as part of the call-to-action text items but I'm not sure if that flows so well as the header/title for the popover. That title appears in two places:

image

image

File: src/app/Topology/Shared/utils.tsx (hunk 2 above)
Comment (andrewazores) [reply src/app/Topology/Shared/utils.tsx:56]: That is there, just a few lines further down: https://github.com/cryostatio/cryostat-web/pull/1119/files/27c2cbc950bb3f0a6e479e64c6d15a120e727e2f#diff-4e598093f82da69d519e2b5a8983e8fd775537388e1cf781f4b3ddb4a5a194adR61

**Review by mwangggg — APPROVED** [review]
looks good to me

**Pull request merged**